src/Symfony/Bundle/DebugBundle/DependencyInjection/Configuration.php

```diff
@@ -26,7 +26,9 @@ public function getConfigTreeBuilder(): TreeBuilder
         $treeBuilder = new TreeBuilder('debug');
 
         $rootNode = $treeBuilder->getRootNode();
-        $rootNode->children()
+        $rootNode
+            ->documentationUri('https://symfony.com/doc/current/reference/configuration/debug.html')
```

Comment:
Am I wrong or should this include the real version to the docs then? Otherwise in a year 6.4 would link to 8.x docs which could have other options?

Reply:
In order to get the number automatically up-to-date, this can use either `Composer\InstalledVersions::getVersion('symfony/debug-bundle')` or `Symfony\Component\HttpKernel\Kernel::VERSION` (but this is not the same package).
In both cases we need to split after the minor version number.

Reply:
I wondered if giving the "current version" page could be acceptable, then the user chooses the version on the site. This would avoid having to maintain the version with each release. I'll take a look at the Composer package and see what's possible

Reply:
It now extracts major+minor and fallbacks on "current" if it's not available for any reason.